enzyme/tools/enzyme-tblgen/datastructures.cpp

```diff
@@ -217,15 +217,15 @@ void fillArgs(const Record *r, SmallVector<std::string, 6> &args,
   assert(argNameToPos.size() == numArgs);
 }
 
-void fillArgUserMap(SmallVector<Rule, 3> &rules,
-                    const SmallVector<std::string, 6> &nameVec,
-                    const SmallVector<size_t, 4> &activeArgs,
+void fillArgUserMap(SmallVectorImpl<Rule> &rules,
+                    ArrayRef<StringRef> nameVec,
```

Comment:
@tgymnich didn't got `ArrayRef<StringRef> nameVec,` to compile, used ArrayRef<std::string> instead,
feel free to fix it in another PR.

Reply:
@ZuseZ4 you are right ArrayRef<StringRef> won't work here.